sql/create/release-1.0.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -193,8 +193,8 @@ CREATE TABLE `t_escheduler_project` (
   `desc` varchar(200) DEFAULT NULL COMMENT 'project description',
   `user_id` int(11) DEFAULT NULL COMMENT 'creator id',
   `flag` tinyint(4) DEFAULT '1' COMMENT '0 not available, 1 available',
-  `create_time` datetime DEFAULT CURRENT_TIMESTAMP COMMENT 'create time',
-  `update_time` datetime DEFAULT CURRENT_TIMESTAMP COMMENT 'update time',
```

Comment:
Hi, thank you very much for your contribution, but I think "`create_time` datetime DEFAULT CURRENT_TIMESTAMP" is better

Reply:
don't change this file, it belongs to release 1.0.0 version, just used for update